addons/account_edi_ubl/data/ubl_templates.xml

```diff
@@ -44,9 +44,9 @@
                         <cbc:Name t-esc="partner.country_id.name"/>
                     </cac:Country>
                 </cac:PostalAddress>
-                <cac:PartyTaxScheme t-if="partner.vat">
-                    <cbc:RegistrationName t-esc="partner.name"/>
-                    <cbc:CompanyID t-esc="partner.vat"/>
+                <cac:PartyTaxScheme t-if="partner.commercial_partner.vat">
```

Comment:
shouldn't it be `partner.commercial_partner_id.vat`

Reply:
Indeed, mb :+1: